<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Network_disk</class>
 <widget class="QWidget" name="Network_disk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="ElaIconButton" name="back_pushButton">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>返回</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ElaIconButton" name="createDir_pushButton">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>新建文件夹</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ElaIconButton" name="upload_pushButton">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>上传</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ElaIconButton" name="download_pushButton">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>下载</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ElaIconButton" name="shared_pushButton">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>分享文件</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>526</width>
                 <height>17</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="ElaIconButton" name="flush_pushButton">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>刷新</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ElaText" name="label">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>我的网盘</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="scrolltext" name="Tips_widget" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>60</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ElaTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElaText</class>
   <extends>QLabel</extends>
   <header>elatext.h</header>
  </customwidget>
  <customwidget>
   <class>scrolltext</class>
   <extends>QWidget</extends>
   <header>scrolltext.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElaIconButton</class>
   <extends>QPushButton</extends>
   <header>elaiconbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ElaTableView</class>
   <extends>QTableView</extends>
   <header>elatableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
